src/render.rs

```diff
@@ -230,14 +262,17 @@ impl Render for Filter {
                 }
             }
             FilterType::AddSlashes => match left {
-                Some(content) => Some(Content::String(Cow::Owned(
-                    content
-                        .render(context)?
-                        .replace(r"\", r"\\")
-                        .replace("\"", "\\\"")
-                        .replace("'", r"\'"),
-                ))),
-                None => Some(Content::String(Cow::Borrowed(""))),
+                Some(content) => content
+                    .render(context)?
+                    .replace(r"\", r"\\")
+                    .replace("\"", "\\\"")
+                    .replace("'", r"\'")
+                    .into_content(),
```

Comment:
The idea here with `into_content` was to make this easier to read, and also easier to turn something into the content that we expect.

Reply:
Now considering that perhaps we should name this `resolve` instead, and implement the render trait for those types.  Let me know what you think.

Reply:
I'd be interested to see how that looks, but perhaps that's best tried in a follow-up PR.

Reply:
Yeah, I'm not entirely certain if it's a good idea or not. Having naming consistency is worth something though.